### ISSUE TITLE
Using variables in port for SRV records

### ISSUE THREAD

**Issue by Luxaaa** [Domain-Connect/spec#88]
Hey,
I am working for a webhosting company and we are currently integration DomainConnect to improove UX.
In addition to traditional webspaces or virtual servers, we are offering game servers. These do not have a dedicated IP address, but rather run on different ports on the same host system.
We need to set a SRV record for this to work. The problem now is, that according to the DomanConnect specification, variables in SRV ports are not allowed. Is there any plan to make this possible?

Here is a small example of what i want to do:
```json
{
  ...
  "records": [
    {
      "type": "A",
      "host": "@",
      "pointsTo": "%ip%"
    }, {
      "type": "SRV",
      "host": "@",
      "pointsTo": "%ip%",
      "port": "%port%",
      ...
    }
  ]
}
```

best regards
Luca

**Comment by pawel-kow** [comment]
One obvious issue would be for the. field type being now a numeric one - means not possible to carry string for a variable in a valid json.
This can be overcome however - the schema would need to account for such mutability. We had this issue in the past with ttl field being a number in a string field in some cases.

**Comment by pawel-kow** [comment]
Same here, type must remain static. For the rest I only heard some thoughts about TTL that some providers do not allow to set freely. Typically there are limits on min, max or a list of values matched with UI drop down list or so.

**Comment by kerolasa** [comment]
TTL is exceptional.  Perhaps settings values, such as...

https://api.cloudflare.com/dns/domainconnect/v2/kerolasa.iki.fi/settings

...could have DNS Provider specific MinTTL and MaxTTL values, so that service
providers can assess whether values they are about to use are within range
DNS provider wants to support.  For example Cloudflare will never allow zero
TTL, and personally I think anything less than 30 is not great.  At the
other end values greater than a day or so start to be too much of a
liability for users who at times need to get changes done quickly - and it's
a shame if in such situations for some strange reason TTL of 42 weeks is in
use (although it is unlikely to find resolver that would actually cache
that long).

**Comment by pawel-kow** [comment]
Yes, there are many ways to skin this cat. One can also tell that DNS providers do "best effort" when setting the TTL.
This is already happening in an implicit way when onboarding templates I know some providers "normalize" TTL values to the ones supported in their system.

**Comment by pawel-kow** [comment]
Merged into Spec rev 67 ver 2.3
